File: VB/WindowsApplication1/Custom DataNavigator/MyDataNavigatorButtons.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.ViewInfo

Namespace WindowsApplication1
	Public Class MyDataNavigatorButtons
		Inherits DataNavigatorButtons
		Public Sub New(ByVal owner As INavigatorOwner)
			MyBase.New(owner)

		End Sub

		Protected Overrides Function CreateViewInfo() As NavigatorButtonsViewInfo
			Return New MyNavigatorButtonsViewInfo(Me)
		End Function



	End Class
End Namespace

File: CS/WindowsApplication1/Custom DataNavigator/MyDataNavigatorButtons.cs
using System;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;

namespace WindowsApplication1
{
    public class MyDataNavigatorButtons : DataNavigatorButtons
    {
        public MyDataNavigatorButtons(INavigatorOwner owner)
            : base(owner)
        {

        }

        protected override NavigatorButtonsViewInfo CreateViewInfo()
        {
            return new MyNavigatorButtonsViewInfo(this);
        }



    }
}
